internal/backend/runtime/omni/controllers/omni/machine_status_snapshot.go

```diff
@@ -32,17 +32,19 @@ type MachineStatusSnapshotController struct {
 	runner       *task.Runner[snapshot.InfoChan, snapshot.CollectTaskSpec]
 	notifyCh     chan *omni.MachineStatusSnapshot
 	siderolinkCh <-chan *omni.MachineStatusSnapshot
+	powerStageCh <-chan *omni.MachineStatusSnapshot
```

Comment:
I wonder if we could reuse `siderolinkCh` here instead of creating a new channel.

Reply:
I just thought this was cleaner, as they don't come from SideroLink. I think it is more obvious this way that there are 2 different sources. Wdyt?

Reply:
I guess we can keep the two channels

Reply:
yes I think it's ok, `select` works with multiple channels beautifully :)